filters/filebyMailingList

```diff
@@ -0,0 +1,12 @@
+# If the email comes from a valid account in address group, but also is a mailing list
```

Comment:
## If the email comes from a valid account in address group, but also is a mailing list